lib/api.admin.js

```diff
@@ -143,6 +143,20 @@ var importUsers = module.exports.importUsers = function(restCtx, tenantAlias, cs
 };
 
 /**
+ * Get all users for a given tenantAlias through the REST API.
+ *
+ * @param  {RestContext}     adminRestCtx        Standard REST Context object associated to a global or tenant administrator
+ * @param  {String}          tenantAlias         tenantAlias for which you wish to receive users
+ * @param  {Function}        callback            Standard callback function
+ * @param  {Object}          callback.err        Standard error object, if any
+ * @param  {User[]}          callback.response   The users for the given tenantAlias
+ */
+var getAllUsersForTenant = module.exports.getAllUsersForTenant = function(adminRestCtx, tenantAlias, callback) {
+    var url = '/api/tenant/' + RestUtil.encodeURIComponent(tenantAlias) + '/users';
```

Comment:
Let's make it `tenants` as plurals are the [best and common practice](http://www.restapitutorial.com/lessons/restfulresourcenaming.html).